resources/embedded_linux/raspberrypi/template.json

```diff
@@ -0,0 +1,22 @@
+{
+  "templates": [
+      {
+          "description": "",
+          "detail": "Create project that connects to Azure IoT Hub, such as sending sensor data to cloud.",
+          "label": "With Azure IoT Hub",
```

Comment:
TODO: @dilin-MS 
Switch the `With Azure IoT Hub` template choice to `Linaro GCC 6.4 & Azure Device SDK`